app/controllers/gobierto_budgets/budgets_controller.rb

```diff
@@ -14,11 +14,6 @@ def index
     @place_budget_lines = GobiertoBudgets::BudgetLine.all(where: { site: current_site, place: @place, level: 1, year: @year, kind: @kind, area_name: @area_name })
     @interesting_expenses = GobiertoBudgets::BudgetLine.all(where: { site: current_site, place: @place, level: 2, year: @year, kind: GobiertoBudgets::BudgetLine::EXPENSE, area_name: @interesting_area })
 
-    @main_budget_lines_summary = GobiertoBudgets::SiteStats.new(site: current_site, year: @year).main_budget_lines_summary
-
-    @any_custom_income_budget_lines  = GobiertoBudgets::BudgetLine.any_data?(site: current_site, kind: GobiertoBudgets::BudgetLine::INCOME,  area: GobiertoBudgets::CustomArea, index: GobiertoBudgets::SearchEngineConfiguration::BudgetLine.index_forecast)
```

Comment:
@ferblape Why remove this?

Reply:
Those variables are not used anymore in the new home, belong to the old home.